x-pack/solutions/observability/packages/kbn-observability-schema/related_dashboards/rest_specs/get_related_dashboards/v1.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { z } from '@kbn/zod';
+import { relatedDashboardSchema } from '../../schema/recommended_dashboard/v1';
+
+export const getRecommendedDashboardsParamsSchema = z.object({
```

Comment:
The file is called `get_related_dashboards` but here we name it `getRecommendedDashboards`. I would prefer to stick with one name, unless there is a distinction between these two terms.

Reply:
💯 in support of fixing the naming. No worries. 

I'll prefer related dashboards since this api will eventually serve both dashboard suggestions and linked dashboards.